pkcs7/src/content_info.rs

```diff
@@ -16,24 +16,24 @@ const CONTENT_TAG: TagNumber = TagNumber::new(0);
 /// ContentInfo ::= SEQUENCE {
 ///   contentType ContentType,
 ///   content
-///     [0] EXPLICIT ANY DEFINED BY contentType OPTIONAL }
+///     [0] EXPLICIT ANY DEFINED BY contentType }
```

Comment:
Can you leave a note above this that the schema comes from RFC5652?

Reply:
There is already one just above, I added a note to document the difference with RFC2315.